classes/Access.php

```diff
@@ -331,14 +331,14 @@ public function removeModuleAccess($idProfile, $idRole)
      * @param int $idProfile Profile ID
      * @param int $idTab Tab ID
      * @param string $lgcAuth Legacy authorization
-     * @param int $enabled Whether access should be granted
-     * @param int $addFromParent Child from parents
+     * @param bool $enabled Whether access should be granted
+     * @param bool $addFromParent Child from parents
      *
      * @return string Whether legacy access has been successfully updated ("ok", "error")
      *
      * @throws Exception
      */
-    public function updateLgcAccess($idProfile, $idTab, $lgcAuth, $enabled, $addFromParent = 0)
+    public function updateLgcAccess($idProfile, $idTab, $lgcAuth, $enabled, $addFromParent = true)
```

Comment:
Please document the BC Break

Reply:
You should read PR description before :roll_eyes:

Reply:
oopsie